builder/js/ncf.js

```diff
@@ -622,15 +622,17 @@ $scope.groupMethodsByCategory = function () {
     return method.agent_support.some(function(agent){return agent === agentSupport});
   }
   $scope.checkFilterMethod = function(method){
+    var deprecatedFilter = $scope.filter.showDeprecated || !method.deprecated;
```

Comment:
Can you add a comment at the begining of this function explaining what is expected ( return true => display the method, return false => filter the user)

Reply:
Sure !!